app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml

```diff
@@ -10,10 +10,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <css src="mage/calendar.css"/>
         <script src="requirejs/require.js"/>
+        <script src="mage/polyfill.js"/>
     </head>
     <body>
         <referenceContainer name="after.body.start">
-            <block class="Magento\Framework\View\Element\Template" name="head.polyfill" as="polyfill" template="Magento_Theme::js/polyfill.phtml" before="-"/>
```

Comment:
Hi @jonathanKingston. Can you please fix it uses the backward compatible way. We can't delete block from the layout. Please look our document for more details about backward compatibility. You can leave the block declaration, and use `<script type="text/x-magento-init">` to initialize js file.